src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  movies: Array<Movie>;
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  const movieCard = movies.map(movie => {
```

Comment:
please try to think of better name for that variable, maybe:
```suggestion
  const mappedMovies= movies.map(movie => {
```